tests/unit/PeopleAndRoles.spec.ts

```diff
@@ -207,79 +199,274 @@ describe('People And Roles component for Correction', () => {
     const wrapper = wrapperFactory()
+
     await wrapper.find(gpAddPerson).trigger('click')
+
+    // verify properties
+    expect(wrapper.vm.$data.activeIndex).toBe(NaN)
     expect(wrapper.vm.$data.isAddingEditingOrgPerson).toBe(true)
+    expect(wrapper.vm.$data.currentOrgPerson.roles).toEqual([{ roleType: 'Partner', appointmentDate: '2022-11-24' }])
+    expect(wrapper.vm.$data.currentOrgPerson.officer.partyType).toBe('person')
+    expect(wrapper.vm.$data.currentOrgPerson.actions).toEqual(['ADDED'])
+
     wrapper.destroy()
   })
 
-  it('sets the data attributes as expected when Add Corporation button is clicked', async () => {
+  it('sets the properties as expected when Add Corporation button is clicked', async () => {
     store.state.stateModel.peopleAndRoles.orgPeople = []
     const wrapper = wrapperFactory()
+
     await wrapper.find(gpAddCorp).trigger('click')
+
+    // verify properties
+    expect(wrapper.vm.$data.activeIndex).toBe(NaN)
     expect(wrapper.vm.$data.isAddingEditingOrgPerson).toBe(true)
+    expect(wrapper.vm.$data.currentOrgPerson.roles).toEqual([{ roleType: 'Partner', appointmentDate: '2022-11-24' }])
+    expect(wrapper.vm.$data.currentOrgPerson.officer.partyType).toBe('organization')
+    expect(wrapper.vm.$data.currentOrgPerson.actions).toEqual(['ADDED'])
+
     wrapper.destroy()
   })
 
   it('shows the add person form when Add Person button is clicked', async () => {
     store.state.stateModel.peopleAndRoles.orgPeople = []
     const wrapper = wrapperFactory()
+
     await wrapper.find(gpAddPerson).trigger('click')
+
     // verify buttons are now disabled
     expect(wrapper.find(gpAddPerson).attributes('disabled')).toBe('disabled')
     expect(wrapper.find(gpAddCorp).attributes('disabled')).toBe('disabled')
+
     // check form
     expect(wrapper.find(orgPersonForm).exists()).toBe(true)
     expect(wrapper.find('.add-person-header').text()).toBe('Add Person')
+
     wrapper.destroy()
   })
 
   it('shows the add corporation form when Add Corporation button is clicked', async () => {
     store.state.stateModel.peopleAndRoles.orgPeople = getOrgList(['partner'])
     const wrapper = wrapperFactory()
     await wrapper.find(gpAddCorp).trigger('click')
+
     // verify buttons are now disabled
     expect(wrapper.find(gpAddPerson).attributes('disabled')).toBe('disabled')
     expect(wrapper.find(gpAddCorp).attributes('disabled')).toBe('disabled')
+
     // check form
     expect(wrapper.find(orgPersonForm).exists()).toBe(true)
     expect(wrapper.find('.add-org-header').text()).toBe('Add Business or Corporation')
+
     wrapper.destroy()
   })
+
+  it('resets state properties correctly', async () => {
+    store.state.stateModel.peopleAndRoles.orgPeople = [
+      {
+        officer: null,
+        roles: [],
+        mailingAddress: null,
+        actions: ['REMOVED', 'REPLACED']
+      }
+    ]
+    const wrapper: any = shallowMount(PeopleAndRoles, {
```

Comment:
Note: shallow mount because I don't care about sub-components. I only care about THIS component the the method under test.